custom_reports/object_management_api/index.yaml

```diff
@@ -19,6 +19,10 @@ paths:
     $ref: ./paths/custom_dimensions_detail.yaml
   /api/analytics/v1/manage/custom-dimensions/slots/:
     $ref: ./paths/custom_dimensions_slots.yaml
+  /api/analytics/v1/manage/product-custom-dimensions/:
```

Comment:
hmm.. we don't have these docs in [reporting](https://github.com/PiwikPRO/PPMS-Analytics-Reporting/blob/8ef256280624177c8589c03a90bd1ce74994d7fb/docs/internal-object-management.yaml), right?

I think we should either go with auto-generated docs or validate their in CI during reporting build against real API - currently it unmaintainable :/
yet another great dev-friday task xD

Reply:
Oh god what a mess. We should move our docs to company one and durig this process we need to clean this up somehow.  This is to important to be pet-project.